Dockerfile

```diff
@@ -1,15 +1,25 @@
-FROM debian:buster-slim AS build
+FROM golang:1.21 as builder
```

Comment:
Thank you so much for the improvements to the Docker files!

 I agree with the use of distroless images, but I think it would be less problematic and more beneficial to download binaries generated by github actions rather than have users build the executables themselves, which would speed up the image generation. What do you think?